### PR TITLE
Publish OpenTelemetry Schema File 1.6.1

### PR DESCRIPTION
**Pull request by carlosalberto**
Also retire the broken 1.6.0 schema.

### CODE REVIEW

**Comment by carlosalberto** [comment]
@open-telemetry/docs-approvers please review this.

**Review by austinlparker — APPROVED** [review]
lgtm

File: static/schemas/1.6.0
Comment (chalin) [inline static/schemas/1.6.0:1]: @austinlparker @carlosalberto - just curious: is there an issue here for folks attempting to access the 1.6.0 and getting a 404?

Comment (chalin) [reply static/schemas/1.6.0:1]: My guess is that we either want to preserve all version files "forever", or introduce appropriate redirect rules. But I could be wrong. @tigrannajaryan WDYT?

Comment (chalin) [reply static/schemas/1.6.0:1]: For my own reference, here's a link to the schema doc I found: https://github.com/open-telemetry/oteps/blob/main/text/0152-telemetry-schemas.md. (I don't know if this OTEP is superseded by some other doc once the proposal is approved.)

Comment (tigrannajaryan) [reply static/schemas/1.6.0:1]: This is `file_format` which is different from and unrelated to schema versions, which are listed in the `versions` section. `file_format` MUST be 1.0.0 for now. `file_format` may change in the future only if  we amend the schema file format itself via a new OTEP which introduces new capabilities in the schema files.

Unlike `file_format`, a new schema version is introduced every time we make a specification release. We are currently on 1.6.1 of specification and of the schema. 1.6.0 was a broken release and because of that it is not listed in the schema files _as if 1.6.0 never happened_.

Comment (chalin) [reply static/schemas/1.6.0:1]: I wasn't commenting on the `file_format` line, I was just attaching the comment to the deleted file `static/schemas/1.6.0` -- sorry if that wasn't clear.

> 1.6.0 was a broken release and because of that it is not listed in the schema files _as if 1.6.0 never happened_.

That answers my question, thanks.

Comment (chalin) [reply static/schemas/1.6.0:1]: Actually, I have a followup question: is there a problem if the content of these **two directories drift apart**?

- https://github.com/open-telemetry/opentelemetry.io/tree/main/static/schemas
- https://github.com/open-telemetry/opentelemetry-specification/tree/main/schemas

Should 1.6.0 be deleted from `opentelemetry-specification/schemas` too, "as if it never happened"?

Comment (tigrannajaryan) [reply static/schemas/1.6.0:1]: >Should 1.6.0 be deleted from opentelemetry-specification/schemas too, "as if it never happened"?

Good catch. Yes, it should be deleted.

The content of the directories should be in sync and I plan to add an action that syncs the content automatically.

Comment (chalin) [reply static/schemas/1.6.0:1]: The ideal situation is to avoid having the same files in two repos. I've opened #709 to track this issue and discussion.

**Pull request merged**
